docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -196,6 +196,20 @@ const theme = createMuitheme({
 });
 ```
 
+### Styles
+
+- Renamed `fade` to `alpha` to better describe the functinality of the utility.
```

Comment:
```suggestion
- Renamed `fade` to `alpha` to better describe the functionality of the utility.
```
Or "purpose"?

Reply:
Will make it the same as in the other PR:

> Renamed `fade` to `alpha` to better describe the it's functionality.

Reply:
```suggestion
- Renamed `fade` to `alpha` to better describe the it's functionality.
```